rocketpy/Rocket.py

```diff
@@ -449,6 +442,59 @@ def addMotor(self, motor, position):
         self.evaluateStaticMargin()
         return None
 
+    def addSurface(self, surface, position):
+        """Adds an aerodynamic surface to the rocket. The aerodynamic surface
+        must be an instance of a class that inherits from the AerodynamicSurface
+
+        Parameters
+        ----------
+        surface : AerodynamicSurface, NoseCone, TrapezoidalFins, EllipticalFins, Tail
+            Aerodynamic surface to be added to the rocket. See AerodynamicSurface class
+            for more information.
+        position : int, float
+            Position, in m, of the aerodynamic surface's center of pressure relative to
+            the user defined rocket coordinate system. See `Rocket.coordinateSystemOrientation`
+            for more information regarding the rocket's coordinate system.
+
+        Returns
+        -------
+        None
+        """
+        self.aerodynamicSurfaces.append(surface)
+        surface.position = position
+        if isinstance(surface, NoseCone):
+            self.nosecone.append(surface)
+        elif isinstance(surface, (TrapezoidalFins, EllipticalFins)):
+            self.fins.append(surface)
+        elif isinstance(surface, Tail):
+            self.tail.append(surface)
```

Comment:
This is fine, but maybe the objects could sort themselves in the background and not having a two instances in different lists. Let me know what you think of this approach:

1 - Make an attribute of `AeroSurfaces` that is a dictionary

```
{
"Nosecone": [],
"Tail": [],
...
}
```

2 - Use the `__repr__` and `__str__` methods of the objects so as to differentiate the types. That way the `addSurface` method is something along the lines:

```
self.aerodynamicSurfaces[str(surface)].append(surface)
```

The `str(surface)` could be an slice of string or an even an `surface.kind`. I understand that this might be trying to reinvent the wheel to turn the code more polymorphic, so balance the drawbacks on what you think is the best way.